### ISSUE TITLE
List of servers received and tried in 2.1.16

### ISSUE THREAD

**Issue by dgilling** [orientechnologies/orientdb#6058]
Why does the client by default attempt to get a list of servers?
While it seems like a nice feature to have client side, doesn't seem to have it on by default with no switch to turn off.

My cluster is already behind my own server-side load balancer and the network is isolated from rest thus if first one fails, it moves to next address which is an unreachable addresses.

Also, I thought this is a 2.2 feature, but I see this behavior in 2.1.16

OStorageRemote:

```
        for (Map<String, Object> listener : listeners) {
          if (((String) listener.get("protocol")).equals("ONetworkProtocolBinary")) {
            String url = (String) listener.get("listen");
            if (!serverURLs.contains(url))
              addHost(url);
          }
        }
```

**Comment by lvca** [comment]
@dgilling I see you called your global cfg NETWORK_BINARY_DNS_FALLBACK_SERVERS_ENABLED, but that piece of code is used with distributed cfg when the list of server is taken, so in case of crash, the client has the list of servers in the cluster. Why disabling it?

**Comment by dgilling** [comment]
My servers are on a network that is isolated/not directly reachable by the network that client lib is running on without going through a load balancer and some ACL checks.
Since I have my own external L4 Load Balancer, I don’t want to depend on any client side or DNS load balancing outside of what I set up.

Let’s say this Load balancer has an IP of 10.0.0.3

When OrientDb loads this “list of servers”, each server will have IP’s such as 192.168.0.3, 192.168.0.4, etc. which are sent to client.
However, the client cannot directly reach 192.168.0.3, etc.

Looking at how this “distributed cfg” ls loaded in OrientDb Client lib, I see it has a list such as the following:

	[10.0.0.3, 192.168.0.3, 192.168.0.4, 192.168…]
	
Since only 10.0.0.3 is accessible, I only want to retry on that first address and never move to the next address in the list, even if 10.0.0.3 fails.
	Later, I may add additional clusters in this list, but I don’t want each individual server in the same cluster broadcasted to the client.

Of coarse, it may be possible to set up each server to have the correct external IP Address/hostname, but that seems like one more thing I need to configure and would rather not.

Let me know if something here seems not correct or out of place.

In general, this setup is what I found best for what I am using for OrientDb: Docker Swarm on Azure, using their new Scale Set VM’s

Derric
